src/lib/rest/rest.cpp

```diff
@@ -177,9 +177,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
     }
     else if (limit == 0)
     {
-      OrionError error(SccBadRequest, std::string("Bad pagination limit: /") + value + "/ [a value of ZERO is unacceptable]");
-      ciP->httpStatusCode = error.code;
-      ciP->answer         = error.smartRender(ciP->apiVersion);
+      ciP->httpStatusCode = SccOk;
       return MHD_YES;
     }
```

Comment:
It would be cleaner to remove the whole `else if (limit == 0)` block.

Reply:
Fix in https://github.com/telefonicaid/fiware-orion/pull/3994/commits/8879a3975cdd73efa2a2d9289e528d99c68ff168